using-mobile-extensions/adobe-experience-platform-messaging/README.md

```diff
@@ -0,0 +1,211 @@
+# Adobe Experience Platform Messaging
+
+The Adobe Experience Platform Messaging mobile extension allows you to send push notification tokens and push notification click-through feedback to the Adobe Experience Platform.
+
+{% hint style="warning" %}
+The Adobe Experience Platform Messaging extension is currently in Beta development.
+{% endhint %}
+
+## Prerequisites
+
+* IMS organization is enabled for messaging.
+* [Mobile Core](../mobile-core/README.md) and [Adobe Experience Platform Edge Network](../adobe-edge/README.md) extensions have been implemented in the app.
+
+## Configure Profile Dataset in Edge Configuration
+1. Follow the [edge configuration](../../getting-started/edge-configuration.md) document to configure edge.
```

Comment:
just a quick hack to store in the back of your mind...for markdown, if you're doing an ordered list, use `1.` for every item.  it auto-increments, and it allows you to modify the list later without having to change all the numbers :)

Reply:
ahhhh make sense